pages/en/lb4/Testing-your-application.md

```diff
@@ -514,19 +544,23 @@ You should have at least one acceptance (end-to-end) test for each of your REST
 
 Here is an example of an acceptance test:
 
+{% include code-caption.html content= "test/acceptance/product.test.ts" %}
 ```ts
-// test/acceptance/product.acceptance.ts
-import {HelloWorldApp} from '../..';
-import {RestBindings, RestServer} from '@loopback/rest';
-import {expect, supertest} from '@loopback/testlab';
+// test/acceptance/product.test.ts
+import {HelloWorldApplication} from '../..';
+import {supertest, expect, createClientForHandler} from '@loopback/testlab';
 import {givenEmptyDatabase, givenProduct} from '../helpers/database.helpers';
+import {RestServer, RestBindings} from '@loopback/rest';
 
 describe('Product (acceptance)', () => {
-  let app: HelloWorldApp;
-  let request: supertest.SuperTest<supertest.Test>;
+  let app: HelloWorldApplication;
+  let client: supertest.SuperTest<supertest.Test>;
```

Comment:
I think you can simplify it by `let client: Client` and import `Client` from `@loopback/testlab`